src/lib.rs

```diff
@@ -362,9 +362,31 @@ fn builder_for_struct(ast: syn::MacroInput) -> quote::Tokens {
         )
     };
 
+    let builder_struct_doc = format!(
+        "Builder for {s}.\n\
+        \n\
+        # Examples\n\
+        \n\
+        ```\n\
```

Comment:
Hm isn't `rust` implicit for Examples? http://rustbyexample.com/meta/doc.html
Or do you generally prefer explicitness here?

Surprisingly I can't find this doc test in the [output](https://travis-ci.org/colin-kiegel/rust-derive-builder/jobs/201316301#L347-L464) of `cargo test`. I will double check if the behaviour in a foreign crate is the same and also if MBEs behave differently. We could resort to some trickery, e.g. put the documentation into `examples/builder_struct_doc_template.rs`, import it via `include_str` and find a workaround for `format!` placeholders (`FOO` instead of `{s}`).